apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/purchase-notice/style.scss

```diff
@@ -0,0 +1,15 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.wpcom-block-editor-purchase-notice {
```

Comment:
@valterlorran where is this class being used?

Reply:
Looks like it's a leftover of this commit? https://github.com/Automattic/wp-calypso/pull/62021/commits/0b59deb08f43344975bf28c83bb7c859a01d92c1

If so, can we remove the entire style file?

Reply:
Hey @mtias, you're right. We can remove it! Thanks for catching it. If you can, please add me to the PR.